src/app-logic/l10n.js

```diff
@@ -25,12 +37,35 @@ export async function fetchMessages(locale: string): Promise<[string, string]> {
  * preferences.
  */
 export function* lazilyParsedBundles(
-  fetchedMessages: Array<[string, string]>
+  fetchedMessages: Array<[string, string]>,
+  pseudoStrategy?: string
 ): Generator<FluentBundle, void, void> {
+  const transform = pseudoStrategy
+    ? PSEUDO_STRATEGIES[pseudoStrategy]
+    : undefined;
   for (const [locale, messages] of fetchedMessages) {
     const resource = new FluentResource(messages);
-    const bundle = new FluentBundle(locale);
+    const bundle = new FluentBundle(locale, {
+      transform,
+    });
     bundle.addResource(resource);
     yield bundle;
   }
 }
+
+/**
+ * From a language string and a pseudoStrategy, this returns which direction the
+ * document should be in.
+ */
+export function getLocaleDirection(
+  language: string,
+  pseudoStrategy?: string
```

Comment:
We can also narrow this type to `'accented' | 'bidi'`. We already check this the if inside, but would be also good to have a type check for this.

Also, maybe we can also a `PseudoStrategy` type for this?

Reply:
Same thing as for `ltr/rtl`, I think there's some value in keeping this explicit.